ui-components/src/page_components/listing/listing_sidebar/Apply.tsx

```diff
@@ -137,15 +137,21 @@ const Apply = (props: ApplyProps) => {
               </a>
             </p>
           ))}
-        {(listing.applicationPickUpAddress || listing.applicationPickUpAddressType) && (
+        {(listing.applicationPickUpAddress ||
+          listing.applicationPickUpAddressType ||
+          listing.buildingAddress) && (
```

Comment:
So we're assuming that the building address is the final fallback, right?

Reply:
Correct.